api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -241,4 +242,24 @@ default String cdrVersionId(CdrVersion cdrVersion) {
 
   @Mapping(target = "terraName", source = "workspace.name")
   TestUserRawlsWorkspace toTestUserRawlsWorkspace(RawlsWorkspaceDetails workspace, String username);
+
+  @Mapping(target = "workspaceId", source = "workspace.id")
+  @Mapping(target = "namespace", source = "workspace.userFacingId")
+  @Mapping(target = "createdBy", source = "workspace.createdBy")
+  @Mapping(target = "name", source = "workspace.displayName")
+  @Mapping(target = "attributes", ignore = true)
+  @Mapping(target = "authorizationDomain", ignore = true)
+  @Mapping(target = "bucketName", ignore = true)
+  @Mapping(target = "isLocked", ignore = true)
+  @Mapping(target = "lastModified", source = "lastUpdatedDate")
+  @Mapping(target = "workflowCollectionName", ignore = true)
+  @Mapping(target = "googleProject", source = "gcpContext.projectId")
+  @Mapping(target = "googleProjectNumber", ignore = true)
```

Comment:
googleProjectNumber is from Rawls I don't think this exists in our DB

Reply:
yes but MapStruct will complain if I don't include it as ignored here